deploy/helm/logs-collector-config.yaml

```diff
@@ -125,7 +130,7 @@ receivers:
         timestamp:
           parse_from: body.time
           layout_type: gotime
-          layout: '2006-01-02T15:04:05.000000000-07:00'
+          layout: '2006-01-02T15:04:05.999999999-07:00'
```

Comment:
Fixing how fraction of second is parsed, I found this issue during the analysis - see https://stackoverflow.com/questions/53837975/time-parse-in-go-with-varying-fractional-second-length for details